polaris-react/src/components/ActionList/components/SearchField/SearchField.scss

```diff
@@ -0,0 +1,236 @@
+    color var(--p-motion-duration-200) var(--p-motion-ease);
+
+  #{$se23} & {
+    padding: 0 $search-icon-width-se23 0 $search-icon-width-se23;
+    color: var(--p-color-text-inverse-subdued);
+    height: $new-input-height-se23;
+    border: var(--p-border-width-1-experimental) solid
+      var(--p-color-border-inverse);
+    border-radius: var(--p-border-radius-2);
+
+    &:hover {
+      border-color: var(--p-color-border-hover);
+    }
+
+    &:active,
+    &:focus {
+      box-shadow: inset 0 0 0 var(--p-border-width-1) var(--p-color-border);
+    }
+  }
+
+  &::placeholder {
+    color: var(--p-color-text);
+
+    #{$se23} & {
+      color: var(--p-color-text-inverse-subdued);
+    }
+  }
+
+  &::-webkit-search-decoration,
+  &::-webkit-search-cancel-button {
+    appearance: none;
+  }
+}
+
+.Icon {
+  position: absolute;
+  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
+  z-index: var(--pc-search-field-icon);
+  top: 50%;
+  left: var(--p-space-2);
+  display: flex;
+  height: $icon-size;
+  pointer-events: none;
+  transform: translateY(-50%);
+
+  #{$se23} & {
+    height: $icon-size-se23;
+    width: $icon-size-se23;
+  }
+
+  svg {
+    fill: var(--p-color-icon);
+
+    #{$se23} & {
+      fill: var(--p-color-icon-subdued);
+    }
+  }
+}
+
+.Clear {
+  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
+  @include focus-ring($size: 'wide');
+  position: relative;
+  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
+  z-index: var(--pc-search-field-action);
+  border: none;
+  appearance: none;
+  background: transparent;
+  padding: var(--p-space-2);
+
+  #{$se23} & {
+    position: absolute;
+    right: var(--p-space-1);
+    padding: var(--p-space-1);
+  }
+
+  svg {
+    fill: var(--p-color-icon);
+
+    #{$se23} & {
+      fill: var(--p-color-icon-subdued);
+    }
+  }
+
+  &:focus,
+  &:hover {
+    outline: none;
+  }
+
+  &:hover svg,
+  &:focus svg {
+    fill: var(--p-color-icon-hover);
+
+    #{$se23} & {
+      fill: var(--p-color-icon-inverse);
+    }
+  }
+
+  &:focus-visible {
+    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
+    @include focus-ring($size: 'wide', $style: 'focused');
+  }
+
+  &:active {
+    svg {
+      fill: var(--p-color-icon-active);
+    }
+
+    &::after {
+      border: none;
+    }
+  }
+}
+
+.Backdrop {
+  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
+  @include focus-ring($border-width: 1px);
+  position: absolute;
+  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
+  z-index: var(--pc-text-field-backdrop);
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background-color: var(--p-color-bg);
+  border: var(--p-border-width-1) solid var(--p-color-border-input);
+  // stylelint-disable-next-line -- hard coded to address accessbility issue https://github.com/Shopify/polaris/issues/7838
+  border-top-color: #898f94;
```

Comment:
I started from the SearchField.scss from the topbar since Textfield and SearchField were both giving me circular dependencies issues. It was using that color 🤷